packages/init/src/index.ts

```diff
@@ -347,9 +337,9 @@ async function installMCPServer(): Promise<{
 	let shouldReconfigure = false;
 
 	if (alreadyConfigured) {
-		log.info("Neon MCP Server is already configured globally");
 		const response = await confirm({
-			message: "Would you like to reconfigure it? (suggested)",
+			message:
+				"Neon MCP Server is already configured. Would you like to reconfigure it? (suggested)?",
```

Comment:
1. Usually, it's a recommended DX to avoid defaulting to "yes" because some people just press "enter/return" without reading and that can often leads to more frustration than simply re-running the command. I trust your take if you think it's still best to default to "yes"

2. Common practice to add the default answer between parenthesis, no need tfor a second `?`. Another common format is `(Y/n)` where capitalized is the default answer.

```suggestion
				"Neon MCP Server is already configured. Would you like to reconfigure it? (yes)",
```